src/bin/topiary/visualise.rs

```diff
@@ -5,12 +5,19 @@ use clap::ValueEnum;
 
 #[derive(ValueEnum, Clone, Copy, Debug)]
 pub enum Visualisation {
+    // JSON is first as it's the default and
+    // we want it displayed first in the help
     Json,
+
+    // All other output formats should be listed
+    // in alphabetical order
+    Dot,
 }
 
 impl From<Visualisation> for topiary::Visualisation {
     fn from(visualisation: Visualisation) -> Self {
         match visualisation {
+            Visualisation::Dot => Self::GraphViz,
```

Comment:
Why two different names?

Reply:
My rationale is that Clap converts the enum member into `lower-kebab-case` and, for the sake of the CLI argument, `dot` is preferable to `graph-viz` (or even `graphviz` or `gv`). However, internally, `Visualisation::GraphViz` is more meaningful than `Visualisation::Dot`.